solidity/dashboard/src/css/chip.less

```diff
@@ -4,6 +4,7 @@
     pending: @pending-color;
     error: @error-color;
     disabled: @color-grey-10;
+    violet-80: #633EB1; // @color-brand-violet-80
```

Comment:
The `violet-80` name doesn't really match the previous ones (emphasis, success, pending). Maybe we should change it for `info` for example? Or something like that?

Reply:
I updated the name according to the [Keep Design System (Sigils component)](https://www.figma.com/file/ARAMFLMOPal0JewjvlZGhX/Token-dApp-Design-System?node-id=2612%3A259).